src/unix/bsd/netbsdlike/netbsd/mod.rs

```diff
@@ -2957,6 +2957,8 @@ extern "C" {
     pub fn flags_to_string(flags: c_ulong, def: *const c_char) -> c_int;
 
     pub fn kinfo_getvmmap(pid: crate::pid_t, cntp: *mut size_t) -> *mut kinfo_vmentry;
+
+    pub fn getdents(fd: c_int, buf: *mut c_char, nbytes: usize) -> c_int;
```

Comment:
This one needs a `link_name` https://github.com/NetBSD/src/blob/bcc4c5ab138642408f293cdfc6eb84dbd7b7b1fb/include/dirent.h#L122